.github/actions/build-dependencies/action.yaml

```diff
@@ -14,12 +14,14 @@ runs:
         go-version: "${{ inputs.go-version }}"
     - name: Install Compilers & Formatters
       run: |
+        sudo add-apt-repository ppa:arighi/virtme-ng -y
```

Comment:
I'm not maintaining this ppa anymore (the version of virtme-ng there is pretty old). You could simply `apt install virtme-ng` if you're using at least an Ubuntu Noble (24.04) image, or, even better, install vng via pip, `pip install --break-system-packages virtme-ng`. This is probably a safer / more reliable approach.

Reply:
@ianchen0119 take a look on my approach: https://github.com/aquasecurity/libbpfgo/pull/478/commits/597d4f1861488b433b137990f9f4b20a89a400ea

It will be merged, so you can rebase on top of it.

Reply:
@geyslan 
It looks good to me.
I will rebase my PR once PR #478 merged.

Thank you.

Reply:
It's in main already. 👍🏻

Reply:
@geyslan 
Done. 👍🏻